readthedocs/rtd_tests/tests/test_api.py

```diff
@@ -390,11 +396,15 @@ def test_response_finished_and_fail(self):
             },
         )
         build = resp.data
-        self.assertEqual(build['state'], 'finished')
-        self.assertEqual(build['error'], '')
-        self.assertEqual(build['exit_code'], 1)
-        self.assertEqual(build['success'], False)
-        self.assertEqual(build['docs_url'], dashboard_url)
+        self.assertEqual(build["state"], "finished")
+        self.assertEqual(build["error"], "")
+        self.assertEqual(build["exit_code"], 1)
+        self.assertEqual(build["success"], False)
+        self.assertTrue(build["docs_url"].endswith(dashboard_url))
+
+        # Validate received URL (fails with ValidationError if incorrect value)
+        url_validator = URLValidator(schemes=["https"])
+        url_validator(build["docs_url"])
```

Comment:
Same here. Let's follow the patterns we already have in place.

Reply:
okay